### PR TITLE
Upgraded compiletest_rs to 0.9.0

### PR DESCRIPTION
**Pull request by oisyn**
The workspace inheritance is in there as well. I will rebase when those are merged in (done).

### CODE REVIEW

File: Cargo.toml
Comment (eddyb) [inline Cargo.toml:54]: You should e.g. "Files: Insert Final Newline" in VSCode, to fix this warning.

**Review by eddyb — CHANGES_REQUESTED** [review]
LGTM modulo the weird final newline thing.

**Comment by oisyn** [comment]
Wait why did android fail all of a sudden?

**Review by eddyb — APPROVED** [review]

**Comment by eddyb** [comment]
> Wait why did android fail all of a sudden?

* (I assume) blocked on #945

**Comment by oisyn** [comment]
> Wait why did android fail all of a sudden?

This is why: #945 🙂

**Pull request merged**
